docs/guides/getting-started-java-spring.md

```diff
@@ -1,15 +1,22 @@
 ---
 id: getting-started-java-spring
-title: Getting started as a Java developer using Spring
-sidebar_label: Getting started as a Java developer using Spring
+title: Get started as a Java developer using Spring
+sidebar_label: Get started with Spring
 description: "Use Spring Boot and the Spring Zeebe SDK to interact with your local Self-Managed Camunda 8 installation."
 keywords: [java, spring, spring zeebe, getting started, user guide, tutorial]
 ---
 
+import SmPrereqs from './react-components/sm-prerequisites.md'
+import Install from './react-components/install-plain-java.md'
+
 <span class="badge badge--beginner">Beginner</span>
-<span class="badge badge--medium">1 hour</span>
+<span class="badge badge--medium">1 hour</span><br /><br />
 
-In this guide, we'll step through using Spring Boot and the [Spring Zeebe SDK](/apis-tools/spring-zeebe-sdk/getting-started.md) with Desktop Modeler to interact with your local Self-Managed Camunda 8 installation.
+:::note
+This tutorial is not intended for production purposes.
+:::
+
+In this guide, we'll step through using Spring Boot and the [Spring Zeebe SDK](/apis-tools/spring-zeebe-sdk/getting-started.md) with Desktop Modeler to interact with your local Self-Managed Camunda 8 installation. You can also follow this guide with SaaS and Web Modeler after you [create an account](https://signup.camunda.com/accounts?utm_source=docs.camunda.io&utm_medium=referral).
```

Comment:
I'm getting mixed feedback on this exact topic. If our goal is to enable SM users better but not alienate SaaS users, what I have in the PR is a compromise of sorts. We could change the wording a bit to say you can do something similar in SaaS and Web Modeler, but you won't find the exact guidance here.

Reply:
@christinaausley since you are online for a bit today, do you have thoughts here you want to add?

Reply:
> Is it very feasible for a SaaS user to come in and use this tutorial? There's some later references to SM, to the Zeebe config, etc - I think it may need a few more tweaks to fulfill this, which we may want to iterate on later.

I've launched [an issue for a second iteration](https://github.com/camunda/developer-experience/issues/337) based on feedback pulled in from @crobbins215, so if we want to move quickly on this PR, I would feed this feedback (pun unintended) into that issue.

In fact, the **Step 1** suggested improvement based on the [feedback](https://docs.google.com/document/d/1Ro5YemR3KJLQT2_TFruStZrUG7XrNAb5dfEnz4gekD8/edit?usp=sharing) is to add a hint with a link to SaaS as an alternative installation method.

Reply:
> We could change the wording a bit to say you can do something similar in SaaS and Web Modeler, but you won't find the exact guidance here.

So, I think all of this to say that if we want to make a quick change here, this would be the solution.

Reply:
Added

> While this guide focuses on Self-Managed, you can do something similar with [SaaS](https://signup.camunda.com/accounts?utm_source=docs.camunda.io&utm_medium=referral).

We also still have the banner with a link to SaaS.